.github/pull_request_template.md

```diff
@@ -11,7 +11,15 @@ Please reference any related feature requests, issues, or PRs here. For example,
 
 ## How is this patch tested?
 
-(Details)
+<!--
+If you're unsure about what to test, where to add tests, or how to run tests, please feel free to ask. We'd be happy to help.
+-->
+- [ ] I've written tests for this PR using an MLflow-adopted testing framework (`pytest` for Python, `JUnit` for Java, `testthat` for R, or `jest` for JavaScript).
+- [ ] I've manually confirmed the proposed changes work properly.
```

Comment:
Could we make it a little bit more clear about when each of these approaches should be done?
I could see users selecting the 2nd box for new features
i.e., 

New Feature: 
[ ] I have added unit tests that ensure this feature works... (the wording you used is good) 
[ ] I have validated that unit tests pass locally for the test module that I have added to (e.g., I added a feature to the tracking server and made sure that the test suite that validates this behavior passes locally - some wording like that)

Bug-fix:
[ ] I have confirmed that existing tests pass locally and that the issue is resolved on a local MLflow environment (or something like that)

Reply:
Thanks for the comment! I think tests are required in both cases.

Reply:
We could do something like this as you suggested, but this makes the pull request template very long.

```
# new feature

- [ ] ...
- [ ] ...

# bug fix

- [ ] ...

# doc update

- [ ] ... 

```

Reply:
> Should we add something about UI updates "I made sure that if I changed anything in the UI that it renders correctly"?

Does `confirmed the proposed feature/bug-fix/change works` include that?